ConsoleApp10/ConsoleApp10/Program.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+class Program
+{
+    static Random rnd = new Random();
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите количество строк массива в пределах от 5 до 20");
+        Console.WriteLine("------------------------------------------------------");
+        var rowNumber = int.Parse(Console.ReadLine());
+        Console.WriteLine("Введите количество столбцов массива в пределах от 5 до 20");
+        Console.WriteLine("---------------------------------------------------------");
+        var columnNumber = int.Parse(Console.ReadLine());
+
+        var MMM = new int[rowNumber, columnNumber];
+        FillRandom(MMM);
+        PrintMatrix(MMM);
+
+        var sr = GetAverageByRow(MMM);
+        for (var i = 0; i < sr.Length; i++)
+            Console.WriteLine($"Строка:{i,2} Среднее: {sr[i]}");
+
+        Console.ReadKey();
+    }
+
+    static void PrintMatrix(int[,] MMM)
+    {
+        for (var i = 0; i < MMM.GetLength(0); i++)
+        {
+            for (var j = 0; j < MMM.GetLength(1); j++)
+                Console.Write($"{MMM[i, j],3} ");
+            Console.WriteLine();
+        }
+
+        Console.WriteLine();
+    }
```

Comment:
Параметр метода принято называть с маленькой буквы
```suggestion
    static void PrintMatrix(int[,] matrix)
    {
        for (var i = 0; i < matrix.GetLength(0); i++)
        {
            for (var j = 0; j < matrix.GetLength(1); j++)
                Console.Write($"{matrix[i, j],3} ");
            Console.WriteLine();
        }

        Console.WriteLine();
    }
```